client/checkout/style.scss

```diff
@@ -9,25 +9,46 @@
 	border: none !important;
 	// some themes might override the color on all `button`s - whose selector has higher specificity and our class selector.
 	background-color: transparent !important;
-	padding: 3px;
+	font-weight: normal;
```

Comment:
Since the markup changed, the styles needed to change as well.
Before, the `button` element didn't have any text, so we didn't really care what it looked like.
Now, we need to ensure the text within the `button` element plays nice across different themes and browsers.

So I reset the `padding`, changed the `font-size` and `font-weight`, ensured the `cursor` was set to `pointer` (I dunno why, but it wasn't before by default), and text color and background color matched the surrounding text.

I tested this across different themes, but I'm not overly concerned about compatibility.
This is only applied when the plugin is in test mode, so in production the whole instructions set is not going to be visible.